### DISCUSSION TITLE
Is the .NET Annotations Lambda Framework still preview?

### DISCUSSION THREAD

**Discussion by Beau-Gosse-dev** [aws/aws-lambda-dotnet#1439] [General]
This blog says preview, but the README doesn't. 

https://aws.amazon.com/blogs/developer/introducing-net-annotations-lambda-framework-preview/ 

https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Annotations/README.md

Is the .NET Annotations Lambda Framework still preview?

**Comment by petarrepac** [comment]
@Beau-Gosse-dev 

although that blog post will soon be 1 year old, it is still in preview yes

**Reply by ashishdhingra** [reply]
@Beau-Gosse-dev / @petarrepac It's GA now. Please refer https://aws.amazon.com/blogs/developer/net-lambda-annotations-framework/.

@github-actions proposed-answer
